pep-0675.rst

```diff
@@ -537,8 +542,8 @@ Backwards Compatibility
 this doesn't require any changes to the Python runtime itself. :pep:`586`
 already backports ``Literal``, so this PEP does not need to change it.
 
-As :pep:`PEP 586 mentions
-<586#backwards-compatibility>`,
+As PEP 586 `mentions
+<https://www.python.org/dev/peps/pep-0586/#backwards-compatibility>`_,
```

Comment:
```suggestion
As PEP 586 :pep:`mentions <586#backwards-compatibility>`,
```

Is there a reason this was reverted? It is simpler and more reliable to use `:pep:` to refer to PEPs rather than hardcoding the full absolute URL, and will allow us to do things like show the PEP's title in the titletext and other things in the future.

Reply:
@CAM-Gerlach can we add a lint check for this? It could probably look for python.org/dev/peps with a whitelist of ones we don't want to change [yet] (a few in 1,12,287 from memory).

A

Reply:
> Is there a reason this was reverted? It is simpler and more reliable to use :pep: to refer to PEPs rather than hardcoding the full absolute URL, and will allow us to do things like show the PEP's title in the titletext and other things in the future

Huh. The official PEP page renders the `:pep:` link correctly but the GitHub "View file" Markdown preview doesn't. I was going by the latter. Will revert back to the `:pep:` version.

Reply:
> can we add a lint check for this? It could probably look for python.org/dev/peps with a whitelist of ones we don't want to change [yet] (a few in 1,12,287 from memory).

Good idea; in work. If/when PEP 676 is accepted and we transition to the Sphinx-based infra, we could also add one for hardcoded links to PEP supplementary files, instead of using `:download:` (or another appropriate role)—unless you have a good way to do this now that works across both the legacy and new build systems and doesn't break when changing the output mode (files/directories/etc).

> Huh. The official PEP page renders the `:pep:` link correctly but the GitHub "View file" Markdown preview doesn't. I was going by the latter. Will revert back to the `:pep:` version.

The GitHub Markdown preview only has _very_ limited compatibility with RST syntax and should never be relied upon; we will have rendered previews on PRs soon assuming PEP 676 is accepted (or possibly before, depending on when @AA-Turner adds it). Until then, please follow the instructions in the Readme/Contributing guide (basically, install the `requirements.txt` then run the `build.py` script for the new build system or the `pep2html.py` one for the old one to build a preview.